app/presenters/article_presenter.rb

```diff
@@ -0,0 +1,19 @@
+class ArticlePresenter < BasePresenter
+  DOWNLOAD_COMPONENT_IDENTIFIER = 'component_download'.freeze
+
+  def download_component
+    view.render('components/download', content: download_content)
```

Comment:
should this be a constant?

Reply:
I usually moved to a constant but I thought for a while is these are usually code from the view inside the presenter. It is just felt awkward to move to a constant a render partial_name.